query/reverse.js

```diff
@@ -30,7 +31,7 @@ function generateQuery( clean ){
 
   // set size
   if( clean.size ){
-    vs.var( 'size', clean.size );
+    vs.var( 'size', calcSize(clean.size));
```

Comment:
good point. would a separate variable work? something like clean.querySize. it might be useful to have both for other middleware.